src/service/chat.js

```diff
@@ -114,15 +114,15 @@ const sendAnonymousDMOtherProfile = async ({
   message
 }) => {
   const payload = {
-    members: [user_id],
+    member: user_id,
     message,
     anon_user_info_emoji_name,
     anon_user_info_emoji_code,
     anon_user_info_color_name,
     anon_user_info_color_code
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The payload for the `anonymousApi.post` method has been changed from an array of members to a single member. This could potentially break the functionality if the backend API still expects an array of members. Please ensure that the backend API has also been updated to handle this change.
```diff
-    members: [user_id],
+    member: user_id,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->